package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@venusprotocol/token-bridge",
+  "version": "1.0.0",
```

Comment:
I would start with 0.0.1, leaving semantic-release to create the first major version. Otherwise, maybe it will fail (not sure)